### PR TITLE
Removing node-config.yaml references from Nodes bucket II

### PR DESCRIPTION
**Pull request by mburke5678**
Follow up to https://github.com/openshift/openshift-docs/pull/13675

### CODE REVIEW

**Comment by mburke5678** [comment]
Jira for followup changes: https://jira.coreos.com/browse/OSDOCS-311
--

File: modules/nodes-cluster-overcommit-node-enforcing.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
     cpu-cfs-quota: <3>
       - "false"
 ----
-<1> Assign a name to CRD.
+<1> Assign a name to CR.
```
Comment (weinliu) [inline modules/nodes-cluster-overcommit-node-enforcing.adoc]: We used "CRD" in modules/nodes-nodes-garbage-collection-configuring.adoc
Do we need to make it  consistent?

File: modules/nodes-cluster-overcommit-node-memory.adoc [hunk 2]
```diff
@@ -13,22 +13,69 @@ from lower OoS classes from using resources requested by pods in higher QoS clas
...
+.. If the label is not present, add a key/value pair:
++
+[source,bash]
+----
+oc label machineconfigpool worker custom-kubelet=small-pods
+---- 
 
+.Procedure
+
+. Create a Custom Resource (CR) for your configuration change.
```
Comment (weinliu) [inline modules/nodes-cluster-overcommit-node-memory.adoc]: We used "CRD" in modules/nodes-nodes-garbage-collection-configuring.adoc
Do we need to make it consistent?

Comment (mburke5678) [reply modules/nodes-cluster-overcommit-node-memory.adoc]: @weinliu Yes. They are both Custom Resources. I updated `modules/nodes-nodes-garbage-collection-configuring.adoc`. https://github.com/openshift/openshift-docs/pull/14580

File: modules/nodes-cluster-overcommit-node-enforcing.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-cluster-overcommit-node-enforcing.adoc]: @weinliu Yes. This is a Custom Resource (CR). The discrepancy was caused by my misunderstanding of the distinction between the CRD and CR. From my understanding, the user should not edit the CRD.

**Pull request merged**